### DISCUSSION TITLE
Integrating external logs to Security Onion

### DISCUSSION THREAD

**Discussion by gchumash** [Security-Onion-Solutions/securityonion#4341] [Unsupported Versions]
How do we integrate logs/detections from passive live traffic analysis engine like extrahop/forescout/flowmon?

Any ideas?

**Comment by presianbg** [comment]
Like your other thread about the anomaly detection integration, please be more specific.
You may ingest all kind of logs in SO, but you have to create parsers from them, if not available.
https://docs.securityonion.net/en/latest/syslog.html?highlight=syslog

**Comment by weslambert** [comment]
We will be addressing some of this in an upcoming release via the use of Filebeat modules, which will make it much easier to bring logs from external sources into Security Onion:

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-modules.html

Stay tuned for more details!

**Comment by TOoSmOotH** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/4434
